packages/server/src/middleware/currentapp.ts

```diff
@@ -15,109 +15,106 @@ import { UserCtx, ContextUser } from "@budibase/types"
 import tracer from "dd-trace"
 
 export default async (ctx: UserCtx, next: any) => {
-  return tracer.trace("currentapp middleware", {}, async span => {
-    // try to get the appID from the request
-    let requestAppId = await utils.getAppIdFromCtx(ctx)
-    if (!requestAppId) {
-      return next()
-    }
+  // try to get the appID from the request
+  let requestAppId = await utils.getAppIdFromCtx(ctx)
+  if (!requestAppId) {
+    return next()
+  }
 
-    if (requestAppId) {
-      span?.addTags({ app_id: requestAppId })
-    }
+  if (requestAppId) {
+    const span = tracer.scope().active()
+    span?.addTags({ app_id: requestAppId })
```

Comment:
```suggestion
    span.addTags({ app_id: requestAppId })
```

Reply:
My editor suggested that span may not be present, so I used the ?

How come this suggestion removes it?

Reply:
Double checked just to be sure 😅 

![CleanShot 2023-12-15 at 13 14 17@2x](https://github.com/Budibase/budibase/assets/338027/f3c1edb0-a255-4a8e-9846-ced451c8d294)